pages/content/amp-dev/documentation/guides-and-tutorials/develop/animations/triggering_css_animations.md

```diff
@@ -48,6 +48,76 @@ To keep you pages lean and speedy, AMP has enforced a 50,000 byte CSS limit in t
 
 ## Adding, Removing, and Toggling Classes
 
+
+[example preview="top-frame" playground="true" imports="amp-bind"]
+```html
+<head>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+    <style amp-custom>
```

Comment:
TBH, this sample should be a standalone sample:

* too much css 
* standalone sample has better discoverability (and SEO)
* the sample does not demonstrate a concept, but provides an actual sample on how to implement an animated hamburger menu

Reply:
I'll open up a separate PR with this as a standalone :D